src/main/java/pages/IssuePage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selectors.byId;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.Selenide.open;
+
+public class IssuePage {
+
+    String commentButtonID = "footer-comment-button";
+    String textTabButtonID = "aui-uid-1";
+    String commentTextAreaXPath = "//*[@id=\"comment-wiki-edit\"]/textarea";
+    String addCommentSubmitButtonID = "issue-comment-add-submit";
+    String issueActionsContainerID = "issue_actions_container";
+    String issueActionsTextXPath = "//*[@id=\"issue_actions_container\"]/div/div/div[@class=\"action-body flooded\"]";

```

Comment:
Reduce please xpath request.

Reply:
Please, replace "" with '
issueActionsTextXPath - you might want to use Xpath Axes like
//*[@id='issue_actions_container']//folowing::*[@class='action-body flooded']";  (this is just an example)